CMakeLists.txt

```diff
@@ -143,22 +150,48 @@ if (BUILD_STATIC)
         z
         ${ODB_PATH}/libodb-sqlite.a
         ${ODB_PATH}/libodb.a
-        sqlite3
+        mod_sqlite3
         ${CMAKE_THREAD_LIBS_INIT}
        )
 else (BUILD_STATIC)
   set (LIBRARIES_TO_LINK
         mod_vcf
         mod_odb
         ${Boost_LIBRARIES}
-        odb-sqlite
-        odb
-        sqlite3
+        bz2
+        z
+        ${ODB_PATH}/libodb-sqlite.a
+        ${ODB_PATH}/libodb.a
+        mod_sqlite3
         ${CMAKE_THREAD_LIBS_INIT}
        )
 endif (BUILD_STATIC)
 
-# Application tests
+
+
+# APPLICATION TESTS
```

Comment:
I don't know why some comments were all in capitals before, but please make them all consistent, with only the first letter being in capitals.